src/stories/commits/CommitList.stories.mdx

```diff
@@ -0,0 +1,18 @@
+import { Meta, Story } from '@storybook/addon-docs/blocks';
+
+<Meta title="Commits/List/Intro" />
+
+# CommitList
+
+**Shows a history of changes on a task.**
+
+## Overview
+
+The **CommitList** component is a data table that shows the GitHub commit
+history of a **task**. This makes it possible to see at a glance how a task has
+changed over time.
+
+Each commit in the list features an id number, an image of the commit author,
```

Comment:
I wonder if we should say their GitHub avatar, which may not be an image of the author depending on what they are using for their avatar.

Reply:
Agreed. We could note that _all_ of this data comes directly from GitHub via a "push" webhook.

Reply:
Updated: https://github.com/oddbird/Metecho/pull/711/commits/a27b2b51ef2cd3526d18bd65576b63611385fe81

Is this more clear @jgerigmeyer  @stacyk ?